server/datastore/mysql/sessions_test.go

```diff
@@ -10,10 +10,24 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestSessionGetters(t *testing.T) {
+func TestSessions(t *testing.T) {
 	ds := CreateMySQLDS(t)
-	defer ds.Close()
 
+	cases := []struct {
+		name string
+		fn   func(t *testing.T, ds *Datastore)
+	}{
+		{"Getters", testSessionsGetters},
```

Comment:
Even though there's only one test in the suite, I refactored it to the new approach so new ones will be easily added.